src/main/resources/static/service-info.json

```
@@ -0,0 +1,77 @@
+{
+  "id": "uk.ebi.eva.eva-seqcol",
+  "name": "Implementation of the GA4GH Sequence collection retrieval API",
+  "type": {
+    "group": "org.ga4gh",
+    "artifact": "seqcol",
+    "version": "0.0.0"
+  },
+  "organization": {
+    "name": "European Variation Archive",
+    "url": "https://www.ebi.ac.uk/eva"
+  },
+  "contactUrl": "mailto:eva-helpdesk@ebi.ac.uk",
+  "documentationUrl": "https://www.ebi.ac.uk/eva/....",
+  "updatedAt": "08-24-2023",
+  "environment": "dev",
+  "version": "tagged version",
+  "seqcol": {
+    "schema": {
+      "description": "A collection of sequences, representing biological sequences including nucleotide or amino acid sequences. For example, a sequence collection may represent the set of chromosomes in a reference genome, the set of transcripts in a transcriptome, a collection of reference microbial sequences of a specific gene, or any other set of sequences.",
+      "type": "object",
+      "properties":{
+        "lengths":{
+          "type": "array",
+          "description": "Number of elements, such as nucleotides or amino acids, in each sequence.",
+          "collated": "true",
+          "items":{
+            "type": "string"
+          }
+        },
+        "names":{
+          "type": "array",
+          "description": "Human-readable identifiers of each sequence, commonly called chromosome names.",
+          "collated": "true",
+          "items":{
+            "type": "string"
+          }
+        },
+        "sequences":{
+          "type": "array",
+          "description": "Digests of sequences computed using the GA4GH digest algorithm (sha512t24u).",
+          "collated": "true",
+          "items":{
+            "type": "string"
+          }
+        },
+        "md5_sequences":{
+          "type": "array",
+          "description": "Digests of sequences computed using md5 digest algorithm.",
+          "collated": "true",
+          "items":{
+            "type": "string"
+          }
+        },
+        "sorted_names_length_pairs":{
```

I made a typo in the issue and we settled on using hyphen instead of underscore. 
Of course we should make sure the names match between the service info and the API but the correct one is the one bellow.
```suggestion
        "sorted-name-length-pairs":{
```

---

actually we decided on underscores, right?